### PR TITLE
3282

### PR DESCRIPTION
**Pull request by bowlofeggs**
This pull request has two commits. The first reorganizes some tests to make the second commit easier to work on. The second fixes a number of issues with the "Push to stable" button.

### CODE REVIEW

File: bodhi/tests/server/test_models.py [hunk 1]
```diff
@@ -710,6 +710,24 @@ def test_mandatory_days_in_testing_status_truthy(self):
...
+
+        # Try putting a - into the name of the release, which should get removed
+        self.obj.name = 'f-11'
+
+        self.assertEqual(self.obj.setting_prefix, 'f11')
+
+    @mock.patch.dict(config, {'f11.status': "It's doing just find, thanks for asking"})
+    def test_setting_status_found(self):
+        """Assert correct return value from the setting_status property when config is found."""
+        self.assertEqual(self.obj.setting_status, "It's doing just find, thanks for asking")
```
Comment (abompard) [inline bodhi/tests/server/test_models.py]: "doing just *find*" ? ;-)
(totally optional and unrelated to this patch ;-) )

**Review by abompard — APPROVED** [review]
Yep, looks fine to me (besides a minor typo)

**Comment by abompard** [comment]
In the CI logs I see `FAIL Required test coverage of 99% not reached. Total coverage: 99.30%`. Weird.

**Comment by abompard** [comment]
Any idea why those unit tests fail?

**Comment by bowlofeggs** [comment]
@abompard I'm not sure - I haven't had time to take a peek yet.

File: bodhi/tests/server/test_models.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/tests/server/test_models.py]: Fixed!

**Pull request merged**